README.md

```diff
@@ -13,17 +12,17 @@ We'll give a quick overview of Node.js and explain non-blocking I/O, and provide
 
 ## Non-Blocking Input/Output
 
-Have you worked with Rails and had to wait while its ActiveRecord was processing some simple database join? Or maybe you remember seeing the fail whale every time Twitter was over the capacity. It happened quite often, maybe every week. Slow systems are not only frustrating to users. They cost money to companies because users leave to other services and products. Slow systems are system that cannot handle the traffic or load properly. The slow is a relative term. One system can perform adequate without comparisons, for example what we consider slow in 2016 is very fast comparing to 2000. The reason can be the use of a bad architecture. With Rails the slowness is due to the high-level of abstraction meaning the Object Relational Mapper (ORM) Active Record and the framework as the whole is doing too many things. It's a good thing for developer because we have to code less, but as the real-life showed—not so good for the systems. Another reason for slowness is the way systems are architectured. 
+Have you worked with Rails and had to wait while its ActiveRecord was processing some simple database join? Or maybe you remember seeing the fail whale every time Twitter was over the capacity. It happened quite often, maybe every week. Slow systems are not only frustrating to users. They cost money to companies because users leave to other services and products. Slow systems are system that cannot handle the traffic or load properly. The slow is a relative term. One system can perform adequate without comparisons, for example what we consider slow in 2016 is very fast comparing to 2000. The reason can be the use of a bad architecture. With Rails the slowness is due to the high-level of abstraction meaning the Object Relational Mapper (ORM) Active Record and the framework as the whole is doing too many things. It's a good thing for developer because we have to code less, but as the real-life showed—not so good for the systems. Another reason for slowness is the way systems are architectured.
 
 ![](fail-whale.gif)
 
-So slow platforms not just annoying to develop with, they cost companies hundreds of thousands of dollars each month in servers and CPUs. What if I told you there is a high performant and scalable system which is extremely efficient yet fun to work with? 
+So slow platforms not just annoying to develop with, they cost companies hundreds of thousands of dollars each month in servers and CPUs. What if I told you there is a high performant and scalable system which is extremely efficient yet fun to work with?
```

Comment:
Remove reference to "I". We like to refer to "we" instead of "I. You could update the sentence with: "What if you found out there was a high performant and scalable system..."